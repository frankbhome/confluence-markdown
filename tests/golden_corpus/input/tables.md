# Tables Example

## Basic Table

| Header 1 | Header 2 | Header 3 |
|----------|----------|----------|
| Cell 1   | Cell 2   | Cell 3   |
| Row 2    | Data     | More     |

## Table with Formatting

| Feature | Status | Notes |
|---------|--------|-------|
| **Headers** | ✅ Complete | Working well |
| `Code` | ✅ Complete | Inline code works |
| [Links](https://example.com) | ⚠️ Partial | Need testing |

## Simple Two-Column Table

| Name | Value |
|------|-------|
| Version | 1.0.0 |
| Author | Test User |
