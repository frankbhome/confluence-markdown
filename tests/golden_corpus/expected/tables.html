<h1>Tables Example</h1>

<h2>Basic Table</h2>

<p>| Header 1 | Header 2 | Header 3 |</p>
<p>|----------|----------|----------|</p>
<p>| Cell 1   | Cell 2   | Cell 3   |</p>
<p>| Row 2    | Data     | More     |</p>

<h2>Table with Formatting</h2>

<p>| Feature | Status | Notes |</p>
<p>|---------|--------|-------|</p>
<p>| <strong>Headers</strong> | ✅ Complete | Working well |</p>
<p>| <code>Code</code> | ✅ Complete | Inline code works |</p>
<p>| <a href="https://example.com">Links</a> | ⚠️ Partial | Need testing |</p>

<h2>Simple Two-Column Table</h2>

<p>| Name | Value |</p>
<p>|------|-------|</p>
<p>| Version | 1.0.0 |</p>
<p>| Author | Test User |</p>
